Added weighted round robin algorithm

hackeys opened the pull request:
Hi Folks , 

I have added new algorithm ie "Weighted Round Robin"

I think it would be very useful for people who wish to use Neutrino for Blue-Green kind of deployments

Commit:
Added weighted round robin algorithm

hackeys committed:
Added weighted round robin algorithm

hackeys committed:
Merge branch 'master' of https://github.com/hackeys/Neutrino

blessonpaul commented:
Build failed.
https://travis-ci.org/eBay/Neutrino/builds/336113228?utm_source=github_status&utm_medium=notification

Can you look into it

Also please add a breif explaination of your logic

hackeys committed:
Consul-template file for Neutrino SLB

Example Consul-template file which can talk to consul and update Neutrino configuration file. It is designed for Multiport configuration

hackeys committed:
Added weighted round robin algorithm

hackeys committed:
Merge https://github.com/hackeys/Neutrino

hackeys committed:
Merge pull request #1 from hackeys/consul-template

Consul-template file for Neutrino SLB
